libr/include/r_types.h

```diff
@@ -210,6 +210,13 @@
   #define FUNC_ATTR_ALWAYS_INLINE
 #endif
 
+/* printf format check attributes */
+#if defined(__clang__) || defined(__GNUC__)
+#define R_PRINTF_CHECK(fmt, dots) __attribute__ ((format (printf, fmt, dots)))
+#else
+#define R_PRINTF_CHECK(format, dots)
```

Comment:
use `fmt`, `dots` as in the previous definition.

Reply:
Fixed